src/apps/content-editor/src/app/components/ContentBreadcrumbs.tsx

```diff
@@ -52,6 +56,12 @@ export const ContentBreadcrumbs = () => {
         parent = null;
       }
     }
+
+    if (!itemZUID && isInMultipageTableView) {
+      // Remove the model as a breadcrumb item when viewing in multipage table view
+      crumbs?.pop();
```

Comment:
@agalin920 Actually that was also my first thought but it turns out the model needs to be a part of the crumbs array at first in order to determine if it has a parent since it needs to go through the whole while loop above to do so, that's why I decided to just leave the it as it is then just pop it out afterwards since we won't be needing it as a breadcrumb item for the table view.